models/api.models.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../db/connection');
+const fs = require('fs/promises')
+
+
+
+exports.returnEndpoints=()=>{
+
+    
+     return fs.readFile("../endpoints.jason",'utf8').then((res)=>{
+        const data= JSON.parse(res);
+        return data;
+    }).catch(()=>{})
+}
```

Comment:
We don't actually need a model here for this ticket, everything can be done in the controller 🙂

Reply:
CHEERS, I WILL EDITE IT